### PR TITLE
Add pageheader

### PR DESCRIPTION
**Pull request by kevinch**
screen shot 2018-07-04 at 11 35 21

Liked to https://github.com/vtex/styleguide/issues/145

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [5.1.0] - 2018-7-3
+## [5.2.0] - 2018-7-04
```
Comment (klzns) [inline CHANGELOG.md]: Faltou um zero

File: react/components/PageHeader/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import Button from '../Button'
+
+class PageHeader extends PureComponent {
+  handleClick = e => {
+    this.props.backClick && this.props.backClick(e)
+  }
+
+  render() {
+    const { backLabel, backUrl } = this.props
+    const backButton = backLabel && backUrl
```
Comment (klzns) [inline react/components/PageHeader/index.js]: backUrl nao ta sendo usado, é isso msm?

File: react/components/PageHeader/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<PageHeader title="Invoices" />
+```
+
+With back link
+
+```js
+<PageHeader
+  title="Invoice"
+  backLabel="Back"
+  backUrl="javascript:;"
```
Comment (klzns) [inline react/components/PageHeader/README.md]: Podia realmente ser uma URL pro exemplo

File: react/components/PageHeader/index.js (hunk 2 above)
Comment (kevinch) [reply react/components/PageHeader/index.js]: True, não usei afinal.

**Comment by cmdalbem** [comment]
There are a couple of problems in the implementation that deviate from the original design:

- [ ] Background color should be light-silver, not near-white (we're killing near-white because of contrast issues)
- [ ] Margins are a little off, should be like this:
image
- [ ] Although the back link could receive a label "Back", it would be more interesting if it was more semantic label, like in the example use cases from the [original issue](https://github.com/vtex/styleguide/issues/145). I think it'd be better not to use "Back" in the Styleguide examples.
- [ ] It'd be interesting if the wether the back links is used or not, the component height and inner margins would stay the same. When overlapping the screenshots of the implemented version we can see that both the height and the margins change:
image

**Comment by cmdalbem** [comment]
There are a couple of problems in the implementation that deviate from the original design. I documented them here: https://github.com/vtex/styleguide/issues/199

**Pull request merged**